docs/amd.rst

```diff
@@ -93,7 +94,9 @@ Arguments
 | ``seed``                                          | A random number generator or seed to use for steps with random sampling.                                        |
 +---------------------------------------------------+-----------------------------------------------------------------------------------------------------------------+
 | ``dv_types``                                      | Dictionary of DV types for multiple DVs (e.g. dv_types = {'target': 2}). Default is None.                       |
-|                                                   | Allowed keys are: 'drug', 'target' and 'complex'. (For TMDD models only)                                        |
+|                                                   | Allowed keys are: 'drug', 'target', 'complex', 'drug_tot' and 'target_tot'. (For TMDD models only)              |
+|                                                   | Note: in R the integers need to be written with an L suffix (i.e. 2L).                                          |
```

Comment:
Hmm pharmr should make this conversion I think instead. I've made an issue here: https://github.com/pharmpy/pharmr/issues/152 I should be able to add this quite fast, so I think this line can be removed (also because this will be the case for all functions that have dicts etc).